tests/Subscriber/ProgressSubscriberTest.php

```diff
@@ -44,6 +44,14 @@ public function testProgress(): void
 
 
 OUT;
+        if ('\\' === DIRECTORY_SEPARATOR) {
```

Comment:
Could you please refactor this conditional assignment so that it is clear to people reading the code later what is actually being tested? That it is a OS test and different OSes have different outputs?

I am not saying the code is wrong but I would argue it will be hard to maintain.

Reply:
extracted a `isWindowsPlatform` method - thanks